src/components/common/CustomMenu.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="menu">
+    <NavigationIcon class="menu__icon" v-s-dropdown-toggle:menu__options />
+    <SDropdown name="menu__options">
+      <ul>
+        <li class="menu__option" v-for="tab in tabs" :key="tab.label">
+          <a :href="tab.path">{{ tab.label }}</a>
+        </li>
+        <hr class="menu__divider" />
+        <li class="menu__option"><a href="/login">Login</a></li>
```

Comment:
here should be `router-link` instead of `a` as it is routing inside the app 👌

Reply:
achhhhh racja